trp_project_scrum/models/project_task_work.py

```diff
@@ -41,3 +41,14 @@ def get_user_related_details(self, user_id):
 
         res = super(ProjectTaskWork, self).get_user_related_details(user_id)
         return res
+
+        @api.model
+        def create(self, vals):
+            user=self.env['res.users'].browse(self._uid)
+            if not user.sudo().employee_ids:
```

Comment:
yes we did not give permissions for anything other than the minimum so "collaborator" has no access to hr.employee .  you can see it is used elswhere to read the information. it is not writing just reading.